tests/test_other.py

```diff
@@ -5517,6 +5517,12 @@ def check(what, args, fail=True, expect=''):
 ''')
     check(EMCONFIGURE, [PYTHON, 'test.py'], expect=shared.LLVM_NM, fail=False)
 
+    create_file('test.c', 'int main() { return 0; }')
+    cache_dir = tempfile.mkdtemp(prefix='emscripten_test_cache_')
```

Comment:
You can just do `mkdir('emscripten_test_cache_')` to create a temp directory withing the test's current working directory (which is it self temporary and deleted automatically).